docs/history.rst

```diff
@@ -51,7 +51,7 @@ Change Log
 
 2.2.0
 ~~~~~
-* Minimum PROJ version is now 6.1.0
+* Minimum PROJ version is now 6.2.0
```

Comment:
Shouldn't this rather be mentioned in a new 2.3.2 / 2.4.0 release note?

Reply:
Yep. I did a search/replace on that ... Thanks for catching that!